AzureDevops.sublime-syntax

```diff
@@ -93,6 +93,14 @@ contexts:
       push: possible-accessor
 
   embedded-bash:
+    - meta_append: true
+    - match: ''
+      embed: scope:source.shell.bash.azure-devops
+      escape: (?!)
+
+  embedded-bash-folded:
     - meta_append: true
     - match: ''
       push: scope:source.shell.bash.azure-devops
+      with_prototype:
```

Comment:
This significantly grows compiled syntax's size.

Syntax cache grows from 0.8 to 6.1MB for the whole package.

Reply:
Not sure about the best way of a proposal, but I've created a PR to illustrate an alternative without such impact on syntax cache sizes.